### PR TITLE
Neocx ta/gui test to navigate to deploy page

### PR DESCRIPTION
**Pull request by marzelus**

### CODE REVIEW

File: gui_tests/part3_opendevicepage_test.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        # check desktop device ressource is avaiable and not deployed currently focus
+        slot_overview_id = ''
+        for slot in dashboard_slotstatus.find_elements(by='xpath', value="div"):
+            if slot.text == 'Desktop Slot':
+                slot_overview_id = slot.get_attribute("data-testing-id")
+                break
+        slot_id = slot_overview_id.replace('device-slot-overview-entry-', '')
+
+        self.click("img[src='assets/img/devices.svg']")
+        self.assert_text_visible('Desktop Slot', f"div[data-testing-id='device_slot-summary-{slot_id}']") # TODO ID noch raus
```
Comment (thbeu) [inline gui_tests/part3_opendevicepage_test.py:17]: TODO remarks are obsolete, aren't they?

File: gui_tests/part3_opendevicepage_test.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+from seleniumbase import BaseCase
+
+
+class BrowserTest(BaseCase): 
+    def test_opendevicepage(self):
+        dashboard_slotstatus = self.get_element("div[class='dashboard-slotstatus-container-slots']")
+
+        # check desktop device ressource is avaiable and not deployed currently focus
```
Comment (thbeu) [inline gui_tests/part3_opendevicepage_test.py:8]:
```suggestion
        # check desktop device ressource is available and not deployed currently focus
```
Check for more

File: gui_tests/part2_openproject_test.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+
+class BrowserTest(BaseCase): 
+    def test_searchandopenproject(self):
+
+        prj_id = self.get_attribute(f"//*[text()[contains(.,'{os.getenv('PROJECTNAME', '')}')]]", "data-testing-id", by='xpath')
+        self.click(f"div[data-testing-id='{prj_id}']")
+        self.assert_text_visible('Project Dashboard', "div[id='projects-id-dashboard']")
+
+        # check elements in sidebar are exists
```
Comment (MGibarian) [inline gui_tests/part2_openproject_test.py:13]:
```suggestion
        # check whether elements in sidebar exist
```

File: gui_tests/part3_opendevicepage_test.py (hunk 2 above)
Comment (MGibarian) [reply gui_tests/part3_opendevicepage_test.py:8]:
```suggestion
        # check if desktop device resource is available and 'not deployed' is currently focused
```

File: gui_tests/part3_opendevicepage_test.py (hunk 1 above)
Comment (marzelus) [reply gui_tests/part3_opendevicepage_test.py:17]: Yes

**Review by MGibarian — CHANGES_REQUESTED** [review]
minor changes hackerman requests

**Review by thbeu — APPROVED** [review]

**Pull request merged**